scene/resources/audio_stream_polyphonic.h

```diff
@@ -54,7 +54,7 @@ class AudioStreamPolyphonic : public AudioStream {
 class AudioStreamPlaybackPolyphonic : public AudioStreamPlayback {
 	GDCLASS(AudioStreamPlaybackPolyphonic, AudioStreamPlayback)
 
-	enum {
+	enum : uint64_t {
```

Comment:
```suggestion
	enum : uint32_t {
```

should be sufficient, as `ID_MASK` equals max unsigned 32

Reply:
@xerothermic06: ping

Reply:
This doesn't really matter I'd say as it's not used for storage anywhere

Reply:
Discarding the suggestion is fine, too.

Though getting this merged would be nice…